applications/crossbar/src/modules/cb_alerts.erl

```diff
@@ -188,12 +195,176 @@ summary(Context) ->
 %%------------------------------------------------------------------------------
 -spec load_summary(cb_context:context()) -> cb_context:context().
 load_summary(Context) ->
-    Context1 = cb_context:set_account_db(Context, ?KZ_ALERTS_DB),
-    crossbar_doc:load_view(?AVAILABLE_LIST
-                          ,[{'keys', view_keys(Context1)}]
-                          ,Context1
-                          ,fun normalize_view_results/2
-                          ).
+    AccountDb = cb_context:account_db(Context),
+    Context1 =
+        crossbar_doc:load_view(?AVAILABLE_LIST
```

Comment:
`crossbar_doc:load_view` is deprecated (or at least for me :) ), so please use `crossbar_view` function form now on!

```erlang
    ViewOptions = [{'keymap', view_keys(Context)}
                  ,{'databases', [?KZ_ALERTS_DB]}
                  ,{'mapper', crossbar_view:map_value_fun()}
                  ],
    crossbar_view:load(Conext, ?AVAILABLE_LIST, ViewOptions)
```